src/app/trainees/trainee-filters/trainee-filters.component.html

```diff
@@ -0,0 +1,25 @@
+<ng-container
+  *ngIf="{
+    countTotal: countTotal$ | async,
+    countUnderNotice: countUnderNotice$ | async,
+    underNotice: underNotice$ | async
+  } as data"
+>
+  <button
```

Comment:
which button is active and which is in-active state wise? mat-tabs or routeActiveLink may be a better way to go for uniformity, but whichever way we go lets have an active and inactive state so users know what filter is active.

Reply:
ah yea good point but am setting the active state via the `mat-primary` class which is bound to `underNotice` true|false from store